src/PerfView/PerfViewData.cs

```diff
@@ -2490,7 +2490,7 @@ private void AddGenericStopEventToRequest(Guid contextId, TraceEvent traceEvent,
         }
         private IisPipelineEvent GetSlowestEvent(IisRequest request)
         {
-            IisPipelineEvent slowestPipelineEvent = new IisPipelineEvent();
+            IisPipelineEvent slowestPipelineEvent = null;
```

Comment:
This looks like it could result in a null reference on line 2523.

Reply:
Fixed!